src/main/webapp/map.html

```diff
@@ -4,6 +4,46 @@
     <title>Sightings</title>
     <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyD7I56a-GC6HwwpF7OIpFa61WR7lqNfzJE"></script>
     <script>
+    /*function createMap(){
+        map = new google.maps.Map(document.getElementById('map'), {
+            center: {lat: 39.9999223, lng: 116.3265296},
+            zoom: 16
+        });
+        var geocoder = new google.maps.Geocoder();
+        document.getElementById('address').addEventListener("change", function() {
+            geocodeAddress(geocoder, map);
+        });
+        map.addListener('click', function(event) {
+            addLandmark(map, event.latLng);
+        });
+    }
+
+    function geocodeAddress(geocoder, resultsMap) {
+        var address = document.getElementById('address').value;
+        geocoder.geocode({'address': address}, function(results, status) {
+          if (status === 'OK') {
+            resultsMap.setCenter(results[0].geometry.location);
+            var marker = new google.maps.Marker({
+              map: resultsMap,
+              position: results[0].geometry.location
+            });
+          } else {
+            alert('Geocode was not successful for the following reason: ' + status);
+          }
+        });
+    }
+    function addLandmark(map, latlng){
+        const marker = new google.maps.Marker({
+            position: latlng,
+            map: map,
+        });
+        var infoWindow = new google.maps.InfoWindow({
+            content: '<h1 id="header">Type to Edit</h1><input type="text" id="input" value="type something..."><button id="button">Save</button>'
+        });
+        marker.addListener('click', function() {
+            infoWindow.open(map, marker);
+        });
+     }*/
```

Comment:
Is this the right way to comment?@Junochiu
Do I need to make other change?

Reply:
Brandy, in general we shouldn't have commented out code. If it's no longer needed, then just delete it. (That's the point of source control... nothing's ever really deleted as it's always saved in some prior version. So there's no need to leave in commented out code.)

Reply:
I think you should coordinate with Nicole on what part of this code we want to keep and how it should be modified.

Reply:
Oops! Sorry that's my bad for not fully understand the point of source control ><

Reply:
I think you can replace my code altogether since the codeU tutorial for map part 3 might be extended from part 2. It will be easier for the third person if we keep to the tutorial and maybe add our customized changes later if we want to use map for other purposes like showing schools.

Reply:
okay. Then, I'll replace Nicole's code and remain the tutorial on part2.